js/app.js

```diff
@@ -1496,7 +1495,7 @@ const main = () => {
           updateTotalPage(data.total);
           $scope.loading = false;
           // scroll back to top when finish searching
-          $('.site-wrapper-innerd').scrollTop(0);
+          document.querySelector('.site-wrapper-innerd').scrollTo({ top: 0 });
```

Comment:
I'm not sure whether this line should be deleted. So maybe handle it in another PR.